benches/iai-callgrind/impl_path_string_for_evaluation_context_iai-callgrind.rs

```diff
@@ -0,0 +1,156 @@
+use std::borrow::Cow;
```

Comment:
I've mentioned it in the previous pr and would like to point it again. It'll be amazing if we could have share the code between different benchmarks runners.

I believe it could be done by using cargo features and share as much code as possible

Reply:
can you please elaborate this

Reply:
could you please specify what is unclear for you?

I can give some suggestion on how to do this, but the exact way to solve it is on you.

Reply:
yeah sure please i just need 1 example

Reply:
@meskill can you check the latest changes

Reply:
I saw you had implemented something to share code between test runners. It didn't work eventually?

Reply:
no it was working but that was too much change in the code might be done in a separate pr